pkg/webhook/conversion/crd_client_config.go

```diff
@@ -23,43 +22,34 @@ type CrdClientConfig struct {
 
 var SupportedConversionReviewVersions = []string{"v1", "v1beta1"}
 
-func (c *CrdClientConfig) Update() error {
+func (c *CrdClientConfig) Update(ctx context.Context) error {
 	var (
 		retryTimeout = 15 * time.Second
+		retryBudget  = 12 // 12 times * 15 sec = 3 min
```

Comment:
I think we can increase it up to 10-15 minutes, to be ensured that all modules are up-to-date